### ISSUE TITLE
ImageryLayer does not accept various ImageryProviders

### ISSUE THREAD

**Issue by rot1024** [CesiumGS/cesium#8938]
In `Cesium.d.ts`, some imagery providers do not satisfy `ImageryProvider` class type constrain.

```ts
const provider = new ArcGisMapServerImageryProvider({
  url: "//services.arcgisonline.com/ArcGIS/rest/services/World_Street_Map/MapServer",
});

const layer = new ImageryLayer(provider, { /* ... */ })
//                             ^^^^^^^^　Error!
// 'ArcGisMapServerImageryProvider' type is missing the following properties from type 'ImageryProvider': defaultNightAlpha, defaultDayAlpha ts(2739)
```

I confirmed `ArcGisMapServerImageryProvider` and `IonImageryProvider` are so, but perhaps there are other providers that is so.

**Comment by mramato** [comment]
Thanks @rot1024, this is already fixed in master and will go out with the 1.70.1 release later this week. See #8908 for the full details/changes if you're interested.

**Comment by rot1024** [comment]
Thank you for quick reply.
